### DISCUSSION TITLE
addFile response result_code

### DISCUSSION THREAD

**Discussion by rrd108** [nav-gov-hu/M2M#53] [Kérdések]
Az `addFile` endpoint per pill `resultCode` helyett `result_code`-t ad vissza.

Látok korábbi discussuionokban is iyet, de a doksi alapján nem ez a jó, illetve a többi végponton is `resultCode` van.

**Comment by Csonti-Piller** [comment]
Köszönjük a visszajelzést, javítjuk (és jelezzük, ha kiadásra kerül a javítás)

**Comment by nanassy72** [comment]
Az addFile esetében egy elírás miatt szerepel resultCode helyett result_code-t. Ezt egyelőre így hagyjuk, és inkább a dokumentációt módosítjuk, hogy ne kelljen emiatt az összes kliensnél változtatni.
